composer.json

```diff
@@ -36,12 +36,12 @@
   },
   "autoload": {
     "psr-4": {
-      "PrestaShop\\Module\\PrestashopCheckout\\": "classes/"
+      "PrestaShop\\Module\\PrestashopCheckout\\": "src"
     },
     "classmap": [
       "ps_checkout.php",
-      "controllers/",
-      "classes/ObjectModel/"
+      "controllers",
+      "classes"
```

Comment:
classes folder is still present ?

Reply:
stay present because the class OrderMatrice has no namespace due to Prestashop

Reply:
ObjectModel cannot be in a namespace because it break internal method of PrestaShop.
We will add a comment in this class to explain it 👍